docs/TuxStores.md

```diff
@@ -0,0 +1,172 @@
+# Tux/Stores
+
+##Table of Contents
+<ol>
+  <li><a href="#Premise">Premise</a></li>
+  <li><a href="#Implementation">Implementation</a>
+    <ol style="list-style-type:upper-alpha">
+      <li><a href="#Requiring-Stores">Requiring Stores</a>
+      <li><a href="#Creating-Stores">Creating Stores</a></li>
+      <ol style="list-style-type:upper-roman">
+        <li><a href="#create-store-example">Create Stores Example</a></li>
+        <li><a href="#Stores-createStore">METHOD Stores.createStore</a></li>
+        <li><a href="#storeReturn">Return OBJECT: store</a>
+          <ol style="list-style-type:lower-alpha">
+            <li><a href="#store-emitChange">METHOD: store.emitChange</a></li>
+            <li><a href="#store-addChangeListener">METHOD: store.addChangeListener</a></li>
+            <li><a href="#store-removeChangeListener">METHOD: store.removeChangeListener</a></li>
+          </ol>
+        </li>
+      </ol>
+      </li>
+    </ol>
+  </li>
+  <li><a href="#TuxStores-Complete-Example">TuxStores Complete Example</a>
+</ol>
+
+##<a id="Premise"></a>Premise [#](#Premise)
+>Tux Stores is a module providing the interface for extending Flux architecture stores with common methods.
+
+##<a id="Implementation"></a>Implementation [#](#Implementation)
+###<a id="Requiring-Stores"></a>Requiring Stores [#](#Requiring-Stores)
+Tux uses Facebook's [Flux](https://facebook.github.io/flux/) architecture and provides a convenience module that adds common methods to stores via a Stores interface. This interface is exposed via:
```

Comment:
Change to ...and provides an interface for creating stores.  This interface is exposed via:...